dcap/types/src/quote3.rs

```diff
@@ -366,6 +366,87 @@ impl From<Error> for quote3_error_t {
 impl ResultFrom<quote3_error_t> for Error {}
 impl ResultInto<Error> for quote3_error_t {}
 
+/// An enumeration of errors which occur when verifying a quote
+///
+/// These errors correspond to error elements of
+/// [`sgx_ql_qv_result_t`](mc_sgx_dcap_sys_types::sgx_ql_qv_result_t).
+#[derive(Copy, Clone, Debug, Display, Eq, Hash, Ord, PartialEq, PartialOrd)]
+#[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
+#[non_exhaustive]
+#[repr(u32)]
+pub enum VerificationError {
```

Comment:
Not sure if we want this in a separate module, it is for to quote version 3...

Reply:
I think it's fine here personally.